deploy/manifests/virtual-kubelet/base/deployment.yaml

```diff
@@ -1,17 +1,30 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: v1
+kind: Service
 metadata:
+  name: vk-service
   labels:
-    app: virtual-kubelet
+    app: vk-service
+spec:
+  ports:
+  - port: 80
+    name: vk-headless-service
+  clusterIP: None
+  selector:
+    app: vk-service
```

Comment:
I think this should reference the app label in the StatefulSet template, shouldn't it? It's app=virtual-kubelet there.

Reply:
Yes you are correct in both of these cases i have updated the service to just use virtual-kubelet as the name.